config/default.yml

```diff
@@ -9,11 +9,17 @@ Capybara:
     - "**/*_steps.rb"
     - "**/features/step_definitions/**/*"
 
+Capybara/AmbiguousClick:
+  Description: Specify the exact target to click on.
+  Enabled: false
```

Comment:
memo) It is not to be enabled by default. It is only a transition destination when `Capybara/ClickLinkOrButtonStyle` is not `EnforcedStyle: strict`.